### ISSUE TITLE
contrast between background and foregroung

### ISSUE THREAD

**Issue by amberrignell** [fac20/Week1-BDIM-DigitalAgency#61]
Almost there ! You got 95% accessibility on the lighthouse report - you're just missing a better contrast ratio between the background and foreground 
![Screenshot 2020-07-17 at 11 44 04](https://user-images.githubusercontent.com/36554605/87773031-d66afb00-c822-11ea-8d48-9924479a1205.png)
